src/components/hv-list/index.js

```diff
@@ -71,14 +71,28 @@ export default class HvList extends PureComponent<HvComponentProps, State> {
     const showScrollIndicator =
       this.props.element.getAttribute('shows-scroll-indicator') !== 'false';
 
+    const itemsContainer =
+      this.props.element.firstChild &&
```

Comment:
maybe I'm missing something, but `<items>` doesn't have to be the first child, right?

Reply:
You're thinking of this kind of markup, or did you have something else in mind?
```
<list>
  <item/>
  <item/>
  <items>
    <item/>
    <item/>
  </items>
</list>
```

Reply:
I've updated to account for this use case, let me know what you think

Reply:
Yes, there can also be many <items> elements if we’ve loaded multiple pages.

Reply:
> there can also be many elements if we’ve loaded multiple pages

Can you clarify this? Does the markup above not accounting for this case?